src/MSBuild.UnitTests/CommandLineSwitches_Tests.cs

```diff
@@ -817,6 +817,50 @@ public void GraphBuildSwitchIdentificationTests()
             Assert.False(emptyParametersAllowed);
         }
 
+        [Fact]
+        public void LowPrioritySwitchIdentificationTests()
+        {
+            CommandLineSwitches.ParameterizedSwitch parameterizedSwitch;
+            string duplicateSwitchErrorMessage;
+            bool multipleParametersAllowed;
+            string missingParametersErrorMessage;
+            bool unquoteParameters;
+            bool emptyParametersAllowed;
+
+            Assert.True(CommandLineSwitches.IsParameterizedSwitch("low", out parameterizedSwitch, out duplicateSwitchErrorMessage, out multipleParametersAllowed, out missingParametersErrorMessage, out unquoteParameters, out emptyParametersAllowed));
+            Assert.Equal(CommandLineSwitches.ParameterizedSwitch.LowPriority, parameterizedSwitch);
+            Assert.Null(duplicateSwitchErrorMessage);
+            Assert.False(multipleParametersAllowed);
+            Assert.Null(missingParametersErrorMessage);
+            Assert.True(unquoteParameters);
+            Assert.False(emptyParametersAllowed);
+
+            Assert.True(CommandLineSwitches.IsParameterizedSwitch("LOW", out parameterizedSwitch, out duplicateSwitchErrorMessage, out multipleParametersAllowed, out missingParametersErrorMessage, out unquoteParameters, out emptyParametersAllowed));
```

Comment:
It would probably be better to switch these to a theory instead of copying ~the same code 4 times. Then you could also use `CommandLineSwitches.IsParameterizedSwitch("low", out CommandLineSwitches.ParameterizedSwitch parameterizedSwitch` and the same for the others to save a few more lines.